lib/config/config.js

```diff
@@ -465,118 +435,6 @@ function truncateOptions(opts) {
     opts.hostname = truncate(String(opts.hostname), INTAKE_STRING_MAX_SIZE);
 }
 
-/**
- * Collects information about the agent, environment and configuration options that are
- * sets from a source (env vars, start options or config file). For each config option it
- * collects
- * - source: where te value came from (environment, start, file)
- * - sourceValue: the raw value provided im the source
- * - normalizedValue: the value normalized by the configuration which is used in the agent
- * - file: if source === 'file' this proprty is defined with the path of the config file
- *
- * @param {Object} config The configuration object with normalized options
- * @param {Array<{ name: string, opts: Object}>} sources The list of sources of options ordered by priority, highest priority first.
- * @param {String | undefined} configFilePath Path of the config file if loaded
- * @returns {Object} Object with agent, environment, and configuration data.
- */
-function getLoggingPreambleData(config, sources, configFilePath) {
```

Comment:
Note for reviewer: this is the function moved and refactored. There are tests that check them implementation in `./tests/logging-preamble.test.js`